packages/contracts/test/contracts/ovm/execution-manager/ExecutionManager.purity-checking.spec.ts

```diff
@@ -1,10 +1,10 @@
 import '../../../setup'
```

Comment:
Just noticed (unrelated to the PR but relevant) -- the filename uses the out of date "purity checking" term and should be changed to "safety checking"

Reply:
Ah yeah I'll change that in the next PR.